source/ceylon/json/Visitor.ceylon

```diff
@@ -0,0 +1,84 @@
+"Callable interface used when traversing JSON data. 
+ 
+ It is the callers responsiblity to ensure the 
+ methods of this interface are called in a sequence 
+ that corresponds to well-formed JSON. 
+ For example, callers should never generate the calling
+ sequence `onKey()`, `onKey()`."
+by("Tom Bentley")
+shared interface Visitor {
```

Comment:
Interesting point. I already find that I often use an import alias `JsonObject=Object` because the collision between `c.l.Object` and `c.j.Object` causes too much confusion for me, so on the one had I see your point. On the other hand `ceylon.json` has gone with these pithy names, so using `Visitor` is in keeping with that style. What else could we call it that would still fit in with the rest of the module?